cordova-js-src/exec.js

```diff
@@ -41,6 +41,20 @@ var execProxy = require('cordova/exec/proxy');
  */
 module.exports = function (success, fail, service, action, args) {
 
+    // Handle the case when we have an old version of splashscreen plugin to avoid the API calls failures
+    if (service === 'SplashScreen') {
+        var pluginsVersions = require("cordova/plugin_list").metadata;
+        var splashscreenVersion = pluginsVersions['cordova-plugin-splashscreen'];
+        var MIN_SPLASHSCREEN_SUPPORTED_VER = 4;
```

Comment:
From this line, should I understand that the next 4.x.x release here https://github.com/apache/cordova-plugin-splashscreen/releases will be fully compatible with windows ?